packages/showcases/core/csf/components/actions/dropdownItem.csf.js

```diff
@@ -0,0 +1,47 @@
+export const parameters = {
+  actions: {
+    handles: ['mouseenter', 'click', 'focusin', 'focusout'],
+  },
+  design: {
+    type: 'figma',
+    url: 'https://www.figma.com/file/zDZIyayUlr1yTWrsi7cFoo/?node-id=2421%3A3',
+  },
+};
+
+export const argTypes = {
+  label: {
+    type: { name: 'string', required: false },
+    description: 'Dropdown item label.',
+    defaultValue: 'Lorem ipsum',
+    control: {
+      type: 'text',
+    },
+  },
+
+  value: {
+    type: { name: 'string' },
+    description: 'banana',
+    defaultValue: 'Default label',
+    control: {
+      type: 'text',
+    },
+  },
+
+  divider: {
+    type: { name: 'boolean' },
+    description: 'Dropdown item divider',
+    defaultValue: false,
+    control: {
+      type: 'boolean',
+    },
+  },
+
+  icon: {
+    type: { name: 'string' },
+    description: 'Dropdown item icon',
+    defaultValue: null,
+    control: {
+      type: 'text',
+    },
+  },
+};
```

Comment:
Is it possible to avoid this? We didn't create a csf for list-item for example.
Also I don't really understand the purpose of having a specific story for `DropdownItem`? I understood for `ListItem` but here I don't really find the value. 
Thanks in advance @jb-thery!